Source/Core/Core/HW/GCMemcard/GCIFile.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Missing the comment that's at the start of many of the source files.

Reply:
Done.